lib/spack/docs/getting_started.rst

```diff
@@ -35,22 +35,22 @@ A build matrix showing which packages are working on which systems is shown belo
       .. code-block:: console
 
          apt update
-         apt install build-essential ca-certificates coreutils curl environment-modules gfortran git gpg lsb-release python3 python3-distutils python3-venv unzip zip
+         apt install bzip2 ca-certificates file g++ gcc gfortran git gnupg gzip lsb-release patch python3 tar unzip xz-utils zstd
 
    .. tab-item:: RHEL
 
       .. code-block:: console
 
          dnf install epel-release
```

Comment:
While this is being evaluated for minimalism: Is EPEL still needed? Originally this was for python3, right?

Reply:
not a rhel user, so idk

Reply:
Neither am I :-)